app/src/components/Map.jsx

```diff
@@ -136,13 +136,18 @@ class Map extends Component {
       return;
     }
     this.state.trackLayer.recalculatePosition();
-
     this.state.trackLayer.drawTile(
       workProps.vesselTrack.seriesGroupData,
       workProps.vesselTrack.selectedSeries,
       workProps.filters,
-      workProps.map.vesselTrackDisplayMode
+      workProps.map.vesselTrackDisplayMode,
+      // I don't understand why this array has way more points than it should
```

Comment:
I need the weight data for track points. How can I get this? `workProps.vesselTrack.seriesGroupData.weight` has like 20000 values while `workProps.vesselTrack.seriesGroupData` has 80 for instance.